src/styles/main.scss

```diff
@@ -0,0 +1,80 @@
+@import "./utils/extends";
+@import "./utils/functions";
+@import "./utils/mixins";
+@import "./utils/variebles";
+
+.watch {
+  position: relative;
+  min-height: getHeight(300px);
+
+  &__minute {
+    @extend %position;
+    top: 200px;
+    width: 15px;
+    height: 100px;
+    background-color: red;
+    border-radius: $border-radius-item;
+    margin: auto;
+
+    &--active {
+      animation: minute 60s linear infinite normal;
+    }
+  }
+
+  @keyframes minute {
+    from {
+      transform-origin: bottom;
+      transform: rotate(0deg);
+    }
+
+    to {
+      transform-origin: bottom;
+      transform: rotate(360deg);
+    }
+  }
+
+  &__circle {
+    @extend %position;
+    top: 290px;
+    @include square($size-watch-circle);
+    background-color: blue;
+    border-radius: 50%;
+    margin: auto;
+  }
+
+  &__second {
+    @extend %position;
+    width: 10px;
+    height: 300px;
+    background-color: blue;
+    border-radius: $border-radius-item;
+    margin: auto;
+
+    &--active {
+      animation: second 1s linear infinite normal;
+    }
+  }
+
+  @keyframes second {
+    from {
+      transform-origin: bottom;
+      transform: rotate(0deg);
+    }
+
+    to {
+      transform-origin: bottom;
+      transform: rotate(360deg);
+    }
+  }
+
+  &__active {
+    @extend %button;
+    left: 10%;
+
+  }
+
+  &__stop {
+    @extend %button;
+    right: 10%;
+  }
+}
```

Comment:
Great job! You made excellent stop-watch! And I hope it wasn't necessary to make it with buttons because I don't know how to make them))
But do you really need 2 keyframes for the same animation? The only thing they're different is duration